src/views/feedback.njk

```diff
@@ -0,0 +1,122 @@
+{% extends 'common/base.njk' %}
```

Comment:
I just ran the format document + precommit and there was a few changes (some tab spacing) - would you be able to run this on your side or i can commit on my side? Happy either way!